test/integration/bootstrap/bootstrap_test.go

```diff
@@ -15,20 +15,114 @@
 package bootstrap
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
 	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
 	"github.com/stretchr/testify/assert"
+
+	"github.com/terraform-google-modules/enterprise-application/test/integration/testutils"
 )
 
 func TestBootstrap(t *testing.T) {
 	bootstrap := tft.NewTFBlueprintTest(t,
-		         tft.WithTFDir("../../../1-bootstrap"),
+		tft.WithTFDir("../../../1-bootstrap"),
 	)
 
 	bootstrap.DefineVerify(func(assert *assert.Assertions) {
 		bootstrap.DefaultVerify(assert)
 
+		projectID := bootstrap.GetStringOutput("project_id")
+		gcloudArgsBucket := gcloud.WithCommonArgs([]string{"--project", projectID, "--json"})
+
+		bucketReposBuild := []string{
+			"mt-build",
+			"af-build",
+			"fs-build",
+		}
```

Comment:
```suggestion
		bucketInfix := []string{
			"mt",
			"af",
			"fs",
		}
```

Reply:
Use the infix instead of the whole "name".
That way you can use this variable for both buckets test: `build` and `log`.
Just need to check inside the foreach where uses the bucket and change to `${infix}-log` or `${infix}-build`